### ISSUE TITLE
Cleanup and papercuts

### ISSUE THREAD

**Issue by TimMcGilly** [RadiationChallengeProjectCAST/RadDataStorage#17]
- [ ] Refactor setupDB.js into better functions.
- [ ] Cleanup Markdown files
- [ ] Remove useless console.logs

**Comment by TimMcGilly** [comment]
Should all errors also be returned as json? Currently most are text.

**Comment by dislocated-su** [comment]
Yes, errors should be returned as json with a code, short error name, and description. (or some alternative of that that provides similar tooling to students)
